requirements.txt

```
@@ -1,4 +1,7 @@
-linode_api4>=5.17.0
+# TODO(Multicluster OBJ): Revert before merging to dev
+# linode-api4>=5.17.0
+git+https://github.com/zliang-akamai/linode_api4-python@zhiwei/bucket-key-update
```

```suggestion
git+https://github.com/linode/linode_api4-python@proj/multicluster-obj
```
It has been merged into Linode's repo

---

Thanks for the heads up!